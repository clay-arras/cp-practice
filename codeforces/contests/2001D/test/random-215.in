1
9
14 9 17 24 14 16 11 11 10
