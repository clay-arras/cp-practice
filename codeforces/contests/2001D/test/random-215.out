7
14 9 17 24 16 11 10
